### PR TITLE
Release 4.5.0

### PR DESCRIPTION
**Pull request by kikettas**
### Description
This release includes:

- Added flash toggle in the camera screen, configurable view `GiniConfiguration.flashToggleEnabled`

### How to test
Nothing to test

### Merging
Automatic

### CODE REVIEW

File: Documentation/source/Changelog.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+4.5.0 (2019-02-28)
+------------------
+
+-   Added flash toggle in the camera screen, configurable view `GiniConfiguration.flashToggleEnabled`.
```
Comment (nsobadjiev) [inline Documentation/source/Changelog.md]: configurable view?
or configurable **via**?

File: GiniVision/Classes/Core/Screens/Camera/Camera.swift [hunk 2]
```diff
@@ -34,9 +36,19 @@ final class Camera: NSObject, CameraProtocol {
     var stillImageOutput: AVCaptureStillImageOutput?
     var didDetectQR: ((GiniQRCodeDocument) -> Void)?
     var giniConfiguration: GiniConfiguration
+    
+    lazy var isFlashSupported: Bool = {
+        #if targetEnvironment(simulator)
```
Comment (nsobadjiev) [inline GiniVision/Classes/Core/Screens/Camera/Camera.swift]: But the simulator doesn't support flash. This default made sense for testing the feature, but now that it's done, maybe not so much.

File: Documentation/source/Changelog.md (hunk 1 above)
Comment (kikettas) [reply Documentation/source/Changelog.md]: Via, for some reason _Atom_ corrected it.

File: GiniVision/Classes/Core/Screens/Camera/Camera.swift (hunk 2 above)
Comment (kikettas) [reply GiniVision/Classes/Core/Screens/Camera/Camera.swift]: The camera button also doesn't make sense and is still there, for testing purposes makes sense.

**Pull request merged**